CHANGELOG.md

```diff
@@ -1,3 +1,122 @@
+# 4.0.0
+This version includes some big changes, so we've decided to make it version 4.0!
+- webdriver-manager is now it's [own NPM](https://www.npmjs.com/package/webdriver-manager), so you
```

Comment:
Nit: There are a few things that change for our users with webdriver-manager being its own node module but this might hint that I would also have to add webdriver-manager as a dependency.